### PR TITLE
[SP-6554] - update to use new oneofs

### PR DESCRIPTION
**Pull request by hagaishapira**

### CODE REVIEW

File: steps/opsgenie/incident/create/manifest.yaml [hunk 1]
```diff
@@ -28,10 +28,18 @@ envs:
...
-    description: "Incident priority,  should be one of 'P1', 'P2', 'P3', 'P4' and 'P5' or empty. (default is 'P3')."
-    type: "string"
+    description: "Incident priority."
+    type: oneof
     required: false
-    example: "P1"
+    default: P3
+    options:
+      values:
+        -
```
Comment (hagaishapira) [inline steps/opsgenie/incident/create/manifest.yaml:37]: @nshem is this empty value supported?

Comment (nshem) [inline steps/opsgenie/incident/create/manifest.yaml:37]:
```suggestion
        - ""
```

File: steps/redis/client-list/manifest.yaml [hunk 2]
```diff
@@ -22,18 +22,40 @@ envs:
...
+        - multi
+        - name
+        - obl
+        - oll
+        - omem
+        - psub
+        - qbuf-free
+        - sub
+        - tot-mem
+        - user
```
Comment (nshem) [inline steps/redis/client-list/manifest.yaml:58]: suggesion:
long lists might be more readable formatted  like so:
      values:  ["addr","age","argv-mem","cmd","db","events".......]

File: steps/coralogix/insight/manifest.yaml [hunk 3]
```diff
@@ -13,9 +13,14 @@ envs:
...
     required: true
   - name: TYPE
-    description: "Type of insight, one of ['new', 'top', 'suspected']."
-    type: "string"
+    description: "Type of insight."
+    type: oneof
     required: true
+    options:
+      values:
+        - new
```
Comment (nshem) [inline steps/coralogix/insight/manifest.yaml:21]: i would add a default even if the first one is the preferred default, its more readable to pips who will edit the manifest after you&me

File: steps/opsgenie/alert/create/manifest.yaml [hunk 4]
```diff
@@ -37,9 +37,15 @@ envs:
...
     required: false
   - name: RESPONDER_TYPE
-    description: "The type of the entity required to respond for this alert. should be one of 'user', 'team', 'escalation', 'schedule'. when specified, this step's variable `RESPONDER_NAME` must also be specified as part of the step."
-    type: "string"
+    description: "The type of the entity required to respond for this alert. When specified, this step's variable `RESPONDER_NAME` must also be specified as part of the step."
+    type: oneof
     required: false
+    options:
+      values:
+        - user
```
Comment (nshem) [inline steps/opsgenie/alert/create/manifest.yaml:45]: default

**Review by nshem — APPROVED** [review]
🎈

**Pull request merged**